awx/api/urls/urls.py

```diff
@@ -2,177 +2,47 @@
 # All Rights Reserved.
 
 from __future__ import absolute_import, unicode_literals
+import importlib
+import inspect
+from os import path, listdir
 from django.urls import include, re_path
 
-from awx import MODE
-from awx.api.generics import LoggedLoginView, LoggedLogoutView
 from awx.api.views.root import (
     ApiRootView,
     ApiV2RootView,
-    ApiV2PingView,
-    ApiV2ConfigView,
-    ApiV2SubscriptionView,
-    ApiV2AttachView,
-)
-from awx.api.views import (
-    AuthView,
-    UserMeList,
-    DashboardView,
-    DashboardJobsGraphView,
-    UnifiedJobTemplateList,
-    UnifiedJobList,
-    HostAnsibleFactsDetail,
-    JobCredentialsList,
-    JobTemplateCredentialsList,
-    SchedulePreview,
-    ScheduleZoneInfo,
-    OAuth2ApplicationList,
-    OAuth2TokenList,
-    ApplicationOAuth2TokenList,
-    OAuth2ApplicationDetail,
-    HostMetricSummaryMonthlyList,
-)
-
-from awx.api.views.bulk import (
-    BulkView,
-    BulkHostCreateView,
-    BulkJobLaunchView,
 )
 
-from awx.api.views.mesh_visualizer import MeshVisualizer
 
-from awx.api.views.metrics import MetricsView
-from awx.api.views.analytics import AWX_ANALYTICS_API_PREFIX
+def extend_urls(urls, dir_name):
+    urls_dir = path.join(path.dirname(inspect.getfile(ApiRootView)), '..', '..', 'api', 'urls', dir_name)
```

Comment:
I'm not in love with finding another known class and then going relative down to api/urls. Maybe we just want to import something from awx.api and go from there? Anyone else have another trick?